functions/tsconfig.json

```diff
@@ -1,17 +1,26 @@
 {
+  "files": ["./__tests__/test.types.d.ts"],
```

Comment:
Does this only compile tests files? Is it not conflicting with 'include' key ?

Reply:
I don't think so but I created a new `tsconfig` file in the directory `__test__`